aQute.libg/test/aQute/libg/command/CommandTest.java

```diff
@@ -2,24 +2,31 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import java.util.Optional;
+
 import org.junit.jupiter.api.Test;
 
 public class CommandTest {
+	private static String java() {
+		return Optional.ofNullable(System.getenv("JAVA_HOME"))
```

Comment:
`aQute.lib.io.IO.JAVA_HOME` is a static holding the location of the JAVA_HOME folder. Please use it. Also, this should be done in a `@BeforeEach` method rather than a static method since this is JUnit 5 test case. The `@BeforeEach` method can set a field with the proper java command path which is then referenced from the test methods.

Reply:
What is the benefit to use @BeforeEach and not @BeforeAll?

Reply:
BeforeEach is instance based and not class based. Unless we have a real need for static initialization, we should avoid it.